src/coreComponents/finiteElement/FiniteElementDiscretization.cpp

```diff
@@ -56,11 +56,8 @@ FiniteElementDiscretization::~FiniteElementDiscretization()
 
 void FiniteElementDiscretization::postProcessInput()
 {
-//  GEOS_ERROR_IF_NE_MSG( m_order, 1, "Higher order finite element spaces are currently not supported." );
-  GEOS_ERROR_IF( m_formulation != "default" && m_formulation != "SEM",
-                 getDataContext() << ": Only standard element formulations and spectral element formulations are currently supported." );
-  GEOS_ERROR_IF_GT_MSG( m_useVem, 1,
-                        getDataContext() << ": The flag useVirtualElements can be either 0 or 1" );
+  GEOS_ERROR_IF( m_useVem < 0 || m_useVem > 1,
+                 getDataContext() << ": The flag useVirtualElements can be either 0 or 1" );
```

Comment:
A good TODO item for someone would be to add proper support for registering `bool` variables in the data repository! We can of course emulate it with
```
enum class Boolean { True, False };
ENUM_STRINGS( Boolean, "true", "false" ); // or some other spelling
```